contracts/DCNTToken.sol

```diff
@@ -15,9 +16,10 @@ contract DCNTToken is ERC20VotesLockable {
     error MintTooSoon();
 
     /// @param _supply amount of tokens to mint at Token Generation Event
-    constructor(uint256 _supply) ERC20("Decent", "DCNT") ERC20Permit("Decent") {
+    constructor(uint256 _supply, address _owner) ERC20("Decent", "DCNT") ERC20Permit("Decent") {
```

Comment:
you could do `Ownable(_owner)` here rather than the transfer line below

Reply:
or just Ownable(msg.sender)

Reply:
Also, I'd assume we'd want the ability to initialize the owner address to one different than what its deployed from? So we could initialize owner to be the DAO address

Reply:
ah cool cool, makes sense